src/Http.php

```diff
@@ -117,7 +119,7 @@ public function request( $method, $url, $headers = array(), $params = array() )
                 $this->curl->setOpt( $ch, CURLOPT_POSTFIELDS, $params );
 
                 if ($this->postDataFormat == 'json') {
-                    array_push( $headers, 'Accept: application/json', 'Content-Type: application/json' );
+                    array_push( $headers, 'Content-Type: application/json' );
```

Comment:
Out of interest, why did you remove the Accept header.  We default to JSON in our response, but isn't it better to be explicit?